ocp_resources/utils.py

```diff
@@ -276,3 +276,39 @@ def _return_resource(_resource, _check_exists, _msg):
                     _check_exists=check_exists,
                     _msg=skip_create_warn_msg,
                 )
+
+
+def add_rules(role_object):
+    """
+    Add a list of rules to a Role / CluterRole object.
+
+    Args:
+        role_object (list): list of dicts of rules. In the dict:
```

Comment:
`role_object (list)` is not a list, it's a resource

Reply:
Ack.